server/historian/packages/historian-base/package.json

```diff
@@ -27,8 +28,9 @@
     "@fluidframework/protocol-definitions": "^0.1017.1",
     "@fluidframework/server-services": "^0.1017.1",
     "@fluidframework/server-services-client": "^0.1017.1",
-    "@fluidframework/server-services-core": "0.1014.0-7764",
+    "@fluidframework/server-services-core": "^0.1017.1",
     "@fluidframework/server-services-utils": "^0.1017.1",
+    "@fluidframework/server-test-utils": "^0.1017.1",
```

Comment:
This should be a devDependency

Reply:
Because I added some test implementations under /test-utils folder which consume components in server-test-utils, this is a required dependency